recipes/soci/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "4.0.1":
+    url: "https://github.com/SOCI/soci/archive/4.0.1.tar.gz"
+    sha256: "fa69347b1a1ef74450c0382b665a67bd6777cc7005bbe09726479625bcf1e29c"
+  "4.0.0":
+    url: "https://github.com/SOCI/soci/archive/4.0.0.tar.gz"
+    sha256: "359b988d8cbe81357835317821919f7e270c0705e41951a92ac1627cb9fe8faf"
```

Comment:
```suggestion
```

Don't add old versions when a new version is available. Except for majors versions.